torch/onnx/utils.py

```diff
@@ -333,7 +334,9 @@ def _model_to_graph(model, args, verbose=False,
     if isinstance(model, torch.jit.ScriptModule):
         assert example_outputs is not None, "example_outputs must be provided when exporting a ScriptModule"
         try:
-            method_graph, params = torch._C._jit_pass_lower_graph(model.forward.graph, model._c)
+            graph = model.forward.graph
+            torch._C._jit_pass_onnx_preinline(graph)
```

Comment:
nit: do we want to move this call before the if (call it for all scripting and tracing).
In this case tracing is not affected, but _jit_pass_onnx_preinline() might evolve to include more cases that affect tracing as well.
I am ok with keeping it the way it is too.

Reply:
Actually we need it for both scripting and tracing (because of nested script modules) It should be called in both barnches